src/test/java/de/tum/cit/aet/artemis/nebula/architecture/NebulaCodeStyleArchitectureTest.java

```diff
@@ -0,0 +1,16 @@
+package de.tum.cit.aet.artemis.nebula.architecture;
+
+import de.tum.cit.aet.artemis.shared.architecture.module.AbstractModuleCodeStyleTest;
+
+class NebulaCodeStyleArchitectureTest extends AbstractModuleCodeStyleTest {
+
+    @Override
+    public String getModulePackage() {
+        return ARTEMIS_PACKAGE + ".nebula";
+    }
+
+    @Override
+    protected int dtoNameEndingThreshold() {
+        return 2;
```

Comment:
```suggestion
        return 0;
```
If you renamed the DTO classes